.drone.yml

```diff
@@ -122,6 +122,33 @@ trigger:
   event:
   - push
 
+
+---
+kind: pipeline
+type: exec
+name: integration-tests-jail-unjail
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+- name: integration-tests
+  commands:
+  - cd integration-tests/jail
+  - ./run.sh
+- name: teardown
+  commands:
+  - cd integration-tests/jail
+  - export JAIL_CLIENT_RPC=9981
+  - export JAIL_CHAIN_RPC=26657
```

Comment:
is this fixed? wouldn't the ports change every time? isn't only `docker-compose down` enough?
also it means that only one test can run at a time? check with @calvinaco on the integration test tear down

Reply:
I believe this is only to make `docker-compose down` happy, when starting, the ports are chosen from free ports.